Makefile

```diff
@@ -56,4 +63,9 @@ docker-build: build
 docker-push:
 	docker push ${IMAGE}
 
+deploy: docker-build docker-push
+	sed 's#\(image:\) .*#\1 $(IMAGE)#' /tmp/deis-$(SHORT_NAME) | kubectl apply --validate=true -f -
```

Comment:
yeah I think it's a relatively new feature and it only works for deployments, IIRC. Saw it at the last sig-apps meeting. Anyways, LGTM!